demo/huge_image_demo.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Should we add an example so that users can learn how to use it?
https://github.com/open-mmlab/mmrotate/blob/7c9a592fdda583f3b6c0d200c14020daca18317b/demo/image_demo.py#L2-L10

Reply:
good suggestion.